### ISSUE TITLE
Importing a certificate manifest crashes the plugin

### ISSUE THREAD

**Issue by dmikalova** [hashicorp/terraform-provider-kubernetes#1435]
### Terraform Version, Provider Version and Kubernetes Version

```
Terraform version: v1.0.8
Kubernetes provider version: v2.5.0
Kubernetes version: 1.21.3-do.0 (DigitalOcean managed)
```

### Affected Resource(s)
kubernetes_manifest with [cert-manager certificate](https://cert-manager.io/docs/usage/certificate/).

### Terraform Configuration Files
```hcl
resource "kubernetes_manifest" "cert_manager_certificate" {
  manifest = {
    apiVersion = "cert-manager.io/v1"
    kind       = "Certificate"
    metadata = {
      name      = digitalocean_record.root.fqdn
      namespace = "default"
    }
    spec = {
      dnsNames = [
        digitalocean_record.root.fqdn,
        digitalocean_record.star.fqdn,
      ]
      issuerRef = {
        name = "digitalocean"
        kind = "ClusterIssuer"
      }
      secretName = "${digitalocean_record.root.fqdn}-cert"
    }
  }
}

```

### Panic Output
``` sh
terraform import 'kubernetes_manifest.cert_manager_certificate' 'apiVersion=cert-manager.io/v1,kind=Certificate,namespace=default,name=e91e63.tech'

kubernetes_manifest.cert_manager_certificate: Importing from ID "apiVersion=cert-manager.io/v1,kind=Certificate,namespace=default,name=e91e63.tech"...

Stack trace from the terraform-provider-kubernetes_v2.5.0_x5 plugin:

panic: can't create a tftypes.Value of type tftypes.Object["computed_fields":tftypes.List[tftypes.String], "field_manager":tftypes.List[tftypes.Object["force_conflicts":tftypes.Bool, "name":tftypes.String]], "manifest":tftypes.DynamicPseudoType, "object":tftypes.DynamicPseudoType, "timeouts":tftypes.List[tftypes.Object["create":tftypes.String, "delete":tftypes.String, "update":tftypes.String]], "wait_for":tftypes.Object["fields":tftypes.Map[tftypes.String]]], required attribute "computed_fields" not set

goroutine 124 [running]:
github.com/hashicorp/terraform-plugin-go/tftypes.NewValue(...)
        /opt/teamcity-agent/work/5d79fe75d4460a2f/src/github.com/hashicorp/terraform-provider-kubernetes/vendor/github.com/hashicorp/terraform-plugin-go/tftypes/value.go:267
github.com/hashicorp/terraform-provider-kubernetes/manifest/provider.(*RawProviderServer).ImportResourceState(0xc000e48380, 0x2690388, 0xc000121200, 0xc0013640a0, 0xc000ef85f8, 0x40e101, 0x10)
        /opt/teamcity-agent/work/5d79fe75d4460a2f/src/github.com/hashicorp/terraform-provider-kubernetes/manifest/provider/import.go:152 +0x2e25
github.com/hashicorp/terraform-plugin-mux.SchemaServer.ImportResourceState(0xc000f809c0, 0xc000f809f0, 0xc00106e680, 0x2, 0x2, 0xc000e48200, 0x1, 0x2690388, 0xc000121200, 0xc0013640a0, ...)
        /opt/teamcity-agent/work/5d79fe75d4460a2f/src/github.com/hashicorp/terraform-provider-kubernetes/vendor/github.com/hashicorp/terraform-plugin-mux/schema_server.go:298 +0x98
github.com/hashicorp/terraform-plugin-go/tfprotov5/server.(*server).ImportResourceState(0xc00106e6a0, 0x2690430, 0xc000121200, 0xc00135c0a0, 0xc00106e6a0, 0xc0015fe1b0, 0xc001185ba0)
        /opt/teamcity-agent/work/5d79fe75d4460a2f/src/github.com/hashicorp/terraform-provider-kubernetes/vendor/github.com/hashicorp/terraform-plugin-go/tfprotov5/server/server.go:349 +0xdc
github.com/hashicorp/terraform-plugin-go/tfprotov5/internal/tfplugin5._Provider_ImportResourceState_Handler(0x22032c0, 0xc00106e6a0, 0x2690430, 0xc0015fe1b0, 0xc001362060, 0x0, 0x2690430, 0xc0015fe1b0, 0xc00005e5b0, 0x68)
        /opt/teamcity-agent/work/5d79fe75d4460a2f/src/github.com/hashicorp/terraform-provider-kubernetes/vendor/github.com/hashicorp/terraform-plugin-go/tfprotov5/internal/tfplugin5/tfplugin5_grpc.pb.go:398 +0x214
google.golang.org/grpc.(*Server).processUnaryRPC(0xc0003228c0, 0x26a7ff8, 0xc0004c3980, 0xc000e2e000, 0xc000f80a80, 0x350fb18, 0x0, 0x0, 0x0)
        /opt/teamcity-agent/work/5d79fe75d4460a2f/src/github.com/hashicorp/terraform-provider-kubernetes/vendor/google.golang.org/grpc/server.go:1217 +0x52b
google.golang.org/grpc.(*Server).handleStream(0xc0003228c0, 0x26a7ff8, 0xc0004c3980, 0xc000e2e000, 0x0)
        /opt/teamcity-agent/work/5d79fe75d4460a2f/src/github.com/hashicorp/terraform-provider-kubernetes/vendor/google.golang.org/grpc/server.go:1540 +0xd0c
google.golang.org/grpc.(*Server).serveStreams.func1.2(0xc0002bc620, 0xc0003228c0, 0x26a7ff8, 0xc0004c3980, 0xc000e2e000)
        /opt/teamcity-agent/work/5d79fe75d4460a2f/src/github.com/hashicorp/terraform-provider-kubernetes/vendor/google.golang.org/grpc/server.go:878 +0xab
created by google.golang.org/grpc.(*Server).serveStreams.func1
        /opt/teamcity-agent/work/5d79fe75d4460a2f/src/github.com/hashicorp/terraform-provider-kubernetes/vendor/google.golang.org/grpc/server.go:876 +0x1fd

Error: The terraform-provider-kubernetes_v2.5.0_x5 plugin crashed!

This is always indicative of a bug within the plugin. It would be immensely
helpful if you could report the crash with the plugin's maintainers so that it
can be fixed. The output above should help diagnose the issue.

╷
│ Error: Plugin did not respond
│ 
│ The plugin encountered an error, and failed to respond to the
│ plugin.(*GRPCProvider).ImportResourceState call. The plugin logs may
│ contain more details.
╵
```

### Steps to Reproduce
Create the certificate manifest in one module, then try to import it in another module

### Expected Behavior
The certificate manifest would import properly into the terraform resource.

### Actual Behavior
The plugin crashes.

### Community Note

* Please vote on this issue by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help the community and maintainers prioritize this request
* If you are interested in working on this issue or have submitted a pull request, please leave a comment

**Comment by jrhouston** [comment]
Thanks for opening this @dmikalova – we just merged a fix for this issue that will be going out in the next release, so hang tight. 🤞

**Comment by yannpichon-su** [comment]
Hi @dmikalova, I have the same crash when importing a kubernetes_manifest ressource. Do you have a schedule date for the next release ? Thanks 😊

**Comment by thomas-riccardi** [comment]
FYI it was released yesterday in v2.5.1

**Comment by dmikalova** [comment]
Thanks for the fix - I tested with a `terraform state rm kubernetes_manifest.cert_manager_certificate ; terraform import 'kubernetes_manifest.cert_manager_certificate' apiVersion=cert-manager.io/v1,kind=Certificate,name=e91e63.tech,namespace=default` and  import now works.

However running `terraform plan` shows that it wants to update the manifest with the values that it already has. I'm supposing this is from the plugin not writing that information into the state. Here is the diff that occurs running plan after import:

``` sh
  # kubernetes_manifest.cert_manager_certificate will be updated in-place
  ~ resource "kubernetes_manifest" "cert_manager_certificate" {
      + manifest = {
          + apiVersion = "cert-manager.io/v1"
          + kind       = "Certificate"
          + metadata   = {
              + name      = "e91e63.tech"
              + namespace = "default"
            }
          + spec       = {
              + dnsNames   = [
                  + "e91e63.tech",
                  + "*.e91e63.tech",
                ]
              + issuerRef  = {
                  + kind = "ClusterIssuer"
                  + name = "digitalocean"
                }
              + secretName = "e91e63.tech-cert"
            }
        }
        # (1 unchanged attribute hidden)
    }
```

Running apply works fine and resolves the diff.
